source/use_case_arome_arctic.rst

```diff
@@ -0,0 +1,116 @@
+Arome Arctic NWP model
+""""""""""""""""""""""
+
+..
+
+Use Case Goal
+=============
+
+Run an artic weather model, this is used to create weather forecasts for the artic region as well as used as input data for other products.
+
+
+Actors
+======
+
+.. 
+
+* **Producers:** the Norwegian Meteorological institute (department: Development centre for weather forecasting)
+
+* **Consumers:** :
+  - Arome artic post-production: input for SIMRA model i longyearbyen (a dedicated turbulence model that is used by Avinor)
+  - The oceam model uses it as input data
+  - PhD students / researchers
+  - Yr
+  - Services such as: fiskinfo, barentswatch 
+
+
+At first glance it appears there are mostly advanced and intermediate consumers of this data. However, the question is if this is 
+because the simple users actually use a product derived from this data (or just use Yr), but not the data directly?
+
```

Comment:
I am thinking that there are no simple consumers who would want direct access to arome arctic. The simple use cases are more around "just tell me if will rain tomorrow in longyearbyen, and that would be something like yr. So yeah, I think its correct that a simple user would only access a derived product.